charts/stable/dokuwiki/Chart.yaml

```diff
@@ -22,7 +22,7 @@ home: https://truecharts.org/charts/stable/dokuwiki
 icon: https://truecharts.org/img/hotlink-ok/chart-icons/dokuwiki.png
 keywords:
   - dokuwiki
-kubeVersion: '>=1.24.0-0'
+kubeVersion: ">=1.24.0-0"
```

Comment:
this was already failing on master before the change, I just fixed the linting:
https://github.com/truecharts/charts/actions/runs/8608362869/job/23590673464#step:11:46

others are the same, hope this is ok and doesn't need to be in multiple PR's